docs/react-ui-collection.md

```diff
@@ -975,31 +975,50 @@ import { RangeSlider } from '';
 		trackColor='#B4B4B4'
 		handleColor='#1E90FF'
 		handleBorderColor='#F1AD3D'
+		desktop=''
+		laptop=''
+		laptopL=''
+		mobileL=''
+  	mobileM=''
```

Comment:
@Taimoormk  sir, I didn't understand about this, however, it looks fine in my editor, on all tags